internal/cli/atlas/security/ldap/status.go

```diff
@@ -70,5 +70,9 @@ func StatusBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
 	cmd.Flags().StringVarP(&opts.Output, flag.Output, flag.OutputShort, "", usage.FormatOut)
 
+	cmd.AddCommand(
+		WatchBuilder(),
+	)
+
```

Comment:
[q] Why does the `StatusBuilder` then invoke the `WatchBuilder`? Is this a pattern used in other parts of the cli, as it's a bit confusing to read here.

Reply:
yes, I used the same pattern in `VerifyBuilder`, which calls the `StatusBuilder`. I am not sure if it is used in other part of project though but this is the only way that I found to define `atlas security ldap verify status watch <ID>`, indeed the user has to type `status` to use `watch`.
Happy to discuss this with @gssbzn as well to find a better way to do it

Reply:
Right, the commands are chaining off each other, ok that makes more sense.